docs/backend/portal-actions.md

```diff
@@ -11,3 +11,127 @@ myst:
 
 # Portal Actions
 
+Actions are links or buttons that allow users to perform specific actions on your site.
+These actions can be anything from viewing a content item to editing a page, to accessing a specific feature of your site.
+
+```{note}
+Actions in Plone are provided by action providers.
+There are at least two providers for actions.
+
+-   Portal actions, provided by the actions tool named `portal_actions` in the ZMI
+-   Content type actions defined by the Factory Type Information (FTI) of the Types Tool, and named `portal_types` in the ZMI
+Here only the portal actions are described.
```

Comment:
For legibility. At first glance, it looked like it was part of the preceding list.

```suggestion

Here only the portal actions are described.
```